src/transaction/flashback.c

```diff
@@ -459,6 +459,13 @@ flashback_make_summary_list (THREAD_ENTRY * thread_p, FLASHBACK_SUMMARY_CONTEXT
 		{
 		  memcpy (&classoid, supplement_data, sizeof (OID));
 
+		  if (oid_is_system_class (&classoid))
```

Comment:
How about turning off the 'supplemental_log' in the createdb() using sysprm_set_force()?

Reply:
Oh.. it is really good idea. 
I will apply it then test if it works.

Reply:
it works well, I applied it

Reply:
This fix needs to apply to the 11.1 branch.

And how about leave hear 'assert (oid_is_system_class (&classoid) == false)'?

Reply:
OK, I know what you mean. I will add assert.